### PR TITLE
Purge Pins on r_anal_purge ##anal

### PR DESCRIPTION
**Pull request by thestr4ng3r**
Filling this template is yrotadnam

**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**

see title.

**Test plan**

call `r_anal_purge()`.

### CODE REVIEW

File: libr/anal/anal.c [hunk 1]
```diff
@@ -419,6 +419,8 @@ R_API int r_anal_purge (RAnal *anal) {
 	sdb_reset (anal->sdb_zigns);
```
Comment (trufae) [inline libr/anal/anal.c:419]: r_anal_purge should return void and have no space before (. can you fix this in this same PR?

also rebase and cross the fingers to be green

**Review by trufae — CHANGES_REQUESTED** [review]

**Comment by XVilka** [comment]
Not sure if related:
```

[XX] TIMEOUT db/anal/mach0 aav find xrefs
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'e anal.loads=true
aa;afna;aac;aar;aan;aav
pd 17 @0x8058
' bins/mach0/macho-bash
-- stdout

-- stderr
[ ] Analyze all flags starting with sym. and entry0 (aa)
[
[x] Analyze all flags starting with sym. and entry0 (aa)

-- exit status: -1
```

**Comment by trufae** [comment]
This is another of the bugs i fixed yesterday, there are 3 of those timeouting tests, all of them fixed in my pr. and no, its totally unrelated to the anal purge

File: libr/anal/anal.c (hunk 1 above)
Comment (thestr4ng3r) [reply libr/anal/anal.c:419]: crosses fingered

**Review by XVilka — APPROVED** [review]

**Pull request merged**